web/docusaurus.config.js

```diff
@@ -226,7 +226,7 @@ const config = {
               },
               {
                 label: 'Newsletter',
-                to: '/newsletter',
+                href: 'https://webforms.pipedrive.com/f/clRn2zcF1N5NGHAJ4Rzd3mVU6Xr55uL2Dm3z62Np2KUlq6vxaslf6xQ5Te3P1O1A6T',
```

Comment:
Switching out the link kind of defeats the point of having a single URL we share, which is currently https://vast.io/newsletter. There's a TS page that has the link hardcoded. Ideally that remains the only place in the codebase so that switching it out is not a nightmare.

The good news is that this works as expected:

```
             {
                label: 'Newsletter',
                href: 'https://vast.io/newsletter',
              },
```